src/core.js

```diff
@@ -608,7 +614,11 @@ $.extend( $.validator, {
 				return element.validity.badInput ? false : $element.val();
 			}
 
-			val = $element.val();
+			if ( element.hasAttribute( "contenteditable" ) ) {
```

Comment:
Please add a comment that we use hasAttribute to support IE and mention the version